CHANGELOG.md

```diff
@@ -89,6 +89,8 @@ All notable changes to this project will be documented in this file.
 * ### `niscope` (NI-SCOPE)
     * #### Added
     * #### Changed
+        * `GrpcStubInterpreter` now accepts the `meas_wfm_size` argument when calling `fetch_array_measurement`.
+            * If using NI gRPC Device Server 2023 Q1, `meas_wfm_size` will be ignored and all available samples will be returned.
```

Comment:
This is a bug in specific version(s) of NI gRPC Device Server right?
We should say so – 

> Due to a bug in NI gRPC Device Server 2023 Q1, `meas_wfm_size` parameter is be ignored and all available samples will be returned.

Reply:
Another way to put it is that this feature requires NI gRPC Device Server 2023 Q2 or later. Older versions ignore this parameter and return all available samples.